config/vufind/EDSRecordDataFormatter.ini

```
@@ -0,0 +1,69 @@
+; This file is equivalent to RecordDataFormatter.ini, but is applied to EDS records instead of Solr records.
+; If using this is not sufficient you can also adapt the specs by overriding RecordDataFormatter/Specs/EDS.
+; For more details, see the comments in RecordDataFormatter.ini and documentation in the wiki at:
+; https://vufind.org/wiki/development:architecture:record_data_formatter
+;
+; By default only one multi line field 'Items' is used with getItems of the EDS record as data method. In this
+; context "element" means a single part of an item. E.g. a single author is an element of the item "Authors".
+; Note that this file contains the following additional options that are only used in the
+; template RecordDriver/EDS/data-item.phtml:
+;
+; If a search link should be set on the elements of items (default is false)
+;useSearchLink
+; Type of search link (default is "generic")
+;searchLinkType
+; Prefix for elements of items (default is '')
+;elementPrefix
+; Suffix for elements of items (default is '')
+;elementSuffix
+; Used to separate multiple elements of one item (default is ', ').
+;separator
+; Element limit for each item (default is no limit)
+;limit = 3
+; Translatable abbreviation that is displayed after the elements if the limit is exceeded (default is 'eol_ellipsis')
+;abbreviation
+; Untranslated prefix of the abbreviation (default is '')
+;abbreviationPrefix
+; Untranslated suffix of the abbreviation (default is '')
+;abbreviationSuffix
```

Since these cannot be uncommented here outside of a section, I think maybe reformatting like this would be more readable:

```suggestion
; useSearchLink - If a search link should be set on the elements of items (default is false)
; searchLinkType - Type of search link (default is "generic")
; elementPrefix - Prefix for elements of items (default is '')
; elementSuffix - Suffix for elements of items (default is '')
; separator - Used to separate multiple elements of one item (default is ', ').
; limit - Element limit for each item (default is no limit)
; abbreviation - Translatable abbreviation displayed after elements if limit is exceeded (default is 'eol_ellipsis')
; abbreviationPrefix - Untranslated prefix of the abbreviation (default is '')
; abbreviationSuffix - Untranslated suffix of the abbreviation (default is '')
```

---

Makes sense to me!